packages/documentation/src/stories/foundation/metadata/metadata.docs.mdx

```diff
@@ -0,0 +1,75 @@
+import { Meta, Source } from '@storybook/blocks';
+import MetadataExample from './metadata-example.html?raw';
+import MetadataStories from './metadata.stories';
+import { SourceDarkMode } from '../../../../.storybook/preview';
+
+<Meta of={MetadataStories} />
+
+# Metadata
+
+<div className="lead">
+  Metadata is used to communicate additional information about your website to webbrowser, accessible devices or other applications.
+</div>
+
+The meta-tag has attributes for some general page settings but can also be used to define other metadata using specific names.
+On this page you will find links to other pages that explain some of those names and what they can be used for.
+
+    - [General attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta#attributes)
+    - [Standard names](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name)
+
+<Source code={MetadataExample.slice(236, 511)} dark={SourceDarkMode} language="html" />
```

Comment:
Slicing has to be updated:
<img width="1237" alt="image" src="https://github.com/swisspost/design-system/assets/1659006/4477425a-bdae-424a-90f9-fc99fc241486">

It would be better to have individual snippets that are put together for the final one instead of having to slice the final one. This would allow for changes to the snippets without updating another part of the code.